lib/Varien/File/Csv.php

```diff
@@ -115,7 +116,7 @@ public function saveData($file, $data)
     {
         $fh = fopen($file, 'w');
         foreach ($data as $dataRow) {
-            $this->fputcsv($fh, $dataRow, $this->_delimiter, $this->_enclosure);
+            $this->fputcsv($fh, $dataRow, $this->_delimiter, $this->_enclosure, $this->_escape);
```

Comment:
This seems to be a PHPStan error in my opinion. Starting with PHP 8.4 all 5 parameters are required.

Reply:
Changes in this file can be reverted ... it does not not use php fputcsv(), but Varien_File_Csv::fputcsv()